rtCamp/privacy-sandbox-demos
# Installing tailwind css via node package

## Description

This pull request contains some changes to the structure of the project. The templates were moved to the source folder, which is better for separating the source files and the public content. Outside the structural changes, this pull request also installs tailwind css via node to allow better layout customization, adding the possibility to use a theme file and add custom fonts. At least prettier was included in the project to help keep the code well formatted. 

## Testing Instructions 

1. Clone the repository and checkout `feat/installing-tailwind-css` branch
2. Install all the dependencies with `npm install` or `yarn install`
3. set up the.env file to define the environment
4. start the server using `npm start`
5. Navigate across all the scenarios 
6. Check if all the pages load normally
7. Inspect the code and check that the CSS is being loaded locally from `assets/styles/style.css`
8. Inspect the font family from the header is Open Sans

## Screenshots

**Homepage**

![Screenshot 2023-12-15 at 15 36 14](https://github.com/rtCamp/privacy-sandbox-demos/assets/330792/2973f384-352c-4dcf-b742-91086669dc26)

**Scenario Page**

![Screenshot 2023-12-15 at 15 36 22](https://github.com/rtCamp/privacy-sandbox-demos/assets/330792/da73f449-f5ef-4846-9066-29cc50d29e1f)
